pkg/tsdb/prometheus/resource/resource.go

```diff
@@ -98,3 +99,33 @@ func (r *Resource) Execute(ctx context.Context, req *backend.CallResourceRequest
 
 	return callResponse, err
 }
+
+func (r *Resource) DetectVersion(ctx context.Context, req *backend.CallResourceRequest) (*backend.CallResourceResponse, error) {
+	newReq := &backend.CallResourceRequest{
+		PluginContext: req.PluginContext,
+		Path:          "/api/v1/status/buildinfo",
+	}
+
+	resp, err := r.Execute(ctx, newReq)
+
+	if err != nil && strings.Contains(err.Error(), "empty url") {
+		var emptyBody []byte
+		callResponse := &backend.CallResourceResponse{
+			Status: 200,
+			Body:   emptyBody,
+		}
+
+		return callResponse, nil
+	}
```

Comment:
@kylebrandt I added this to keep the UI from throwing the default error that is thrown for all failed resource requests, is there a better way to do this?